src/scrapers/platformen/mijnbuurtje.py

```diff
@@ -0,0 +1,180 @@
+                       "transform": lambda elem: MijnBuurtjeSource.json_frequency(elem)},
+                   'created_at': {
+                       'xpath': '//div[@class="heading3 heading3--semibold"]/text()',
+                       'transform': lambda text: MijnBuurtjeSource.format_date(text)},
+                   'location': {
+                       'xpath': '//span[@class="meta-item-content" and contains(text(),"Dorp:")]',
+                       'transform': lambda elem: MijnBuurtjeSource.strip_text(elem, "Dorp: ")},
+                   }
+
+    def __init__(self, config: MijnBuurtjeSourceConfig):
+        super().__init__(config)
+        self.item_parser = TreeParser(None, None, self.ITEM_SCHEMA)
+
+    def initiatives(self) -> Generator[InitiativeImport, None, None]:
+        page_counter = 1
+        try:
+            while page_counter < 100:
+                list_page_url = self.config.list_endpoint + f"&page={page_counter}"
+                # schemas: defines fields to be scraped
+                # schema: fieldname:{xpath,all,cast,transform}
+                schemas = {'initiatives':
+                               {'xpath': '//a[@href and contains(@class, "postpreview-content")]',
+                                'all': True,
+                                'transform': lambda elements: self.find_initiative_links(elements)}}
+
+                # initialize TreeParser using url and schemas, returns html tree
+                initiative_parser = TreeParser(list_page_url, None, schemas)
+                if initiative_parser.tree is None:
+                    break
+
+                output = initiative_parser.apply_schemas()
+                for uri in output['initiatives']:
+                    yield InitiativeImport(source_uri=uri)
+
+                page_counter = page_counter + 1
+        except Exception as ex:
+            raise ScrapeException("Error loading list of initiatives") from ex
+
+    def complete(self, initiative: InitiativeImport):
+        try:
+            session_metadata = self.item_parser.get_session_metadata(initiative.source_uri)
+            full_initiative = self.item_parser.apply_schemas(metadata=session_metadata,
+                                                             url=initiative.source_uri)
+            for key, value in full_initiative.items():
+                setattr(initiative, key, value)
+
+            if not initiative.location:
+                initiative.location = self.config.location
+        except Exception as ex:
+            raise ScrapeException(f"Error scraping {initiative.source_uri}") from ex
+
+    def find_initiative_links(self, elements):
+        """
+        filters links. Input seems to be 1 on 1 now. Extracted to method for easier debugging.
+        """
+        initiatives = []
+        for e in elements:
+            links = re.findall(self.config.details_endpoint + "\\d{4,5}", e.attrib['href'])
```

Comment:
Als je in de `__init__` een `re.compile` van deze regex maakt, dan hoeft het niet on runtime voor elk element opnieuw. Zie https://docs.python.org/3/library/re.html#re.compile